bin/i18n/resources/dashboard/marketing_announcements/sync_in.rb

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../../../dashboard/config/environment'
+require_relative '../../../i18n_script_utils'
+require_relative '../../../utils/sync_in_base'
+require_relative '../../../redact_restore_utils'
+require_relative '../../dashboard'
+require_relative '../marketing_announcements'
+
+module I18n
+  module Resources
+    module Dashboard
+      module MarketingAnnouncements
+        class SyncIn < I18n::Utils::SyncInBase
+          def process
+            progress_bar.progress = 50
```

Comment:
If `#prepare_marketing_announcements_data` is the rest 50% of the progress, what is the first 50% then?

Reply:
Good question, this was completely arbitrary. I wanted to give a signal that the process started and is doing something.
Could the progress bar be incremented in the `prepare_marketing_announcements_data` method?

Reply:
Sure, the progress bar be incremented wherever needed

Reply:
But in this case, it will be so fast that nobody ever notices that